torch/_dynamo/decorators.py

```diff
@@ -575,6 +575,18 @@ def mark_dynamic(t, index, *, min=None, max=None):
     4) Attempts to trace this function will explicitly raise. As such, all calls to mark_dynamic must be made
     before torch.compile.
 
+    5) If backend specializations is passed in, we will perform a single generic Dynamo trace followed by
```

Comment:
Do multiple different mark'ed dynamic dims with backend specializations produce a cross product of specialized graphs?